### PR TITLE
Plugin support updated

### PR DESCRIPTION
**Pull request by daxgames**
### Updated from closed PR #849 see here for history.

Added support to extend cmder using plugins in the form of json files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@ build/
 Version v*
 *.bak
 config/user-*
+bin/utils.ps1
```
Comment (DRSDavidSoft) [inline .gitignore]: why is `utils.ps1` ignored? This means we couldn't push the file from local repo

**Comment by DRSDavidSoft** [comment]
@daxgames +1, also @Stanzilla may I suggest adding this to the `v1.4` release project?

File: .gitignore (hunk 1 above)
Comment (daxgames) [reply .gitignore]: because it is put in place by build.ps1 at build time.

Comment (DRSDavidSoft) [reply .gitignore]: @daxgames My bad, I had it confused with `scripts/utils.ps1` instead of `bin/utils.ps1`.

**Comment by daxgames** [comment]
@DRSDavidSoft I like your enthusiasm but this PR has a history.  I like the functionality and I think others will also but the @cmderdev/trusted-contributors had some reservations.

**Comment by DRSDavidSoft** [comment]
@daxgames I just hope those reservations are resolved by now, as I was also perplexed by the unmerged status of the previous one.

**Comment by daxgames** [comment]
bump

**Comment by daxgames** [comment]
Still working on it

**Pull request closed without merging**